src/components/Status/LeadingOption.tsx

```diff
@@ -15,6 +16,9 @@ export type LeadingOptionProps = React.HTMLAttributes<HTMLDivElement> & {
 
 export default React.memo(function LeadingOption({ status, leadingOption, metVP, userChoice }: LeadingOptionProps) {
   const t = useFormatMessage()
+  const [user] = useAuthContext()
+
+  const hideLeadingOption = useMemo(() => !!user && !userChoice, [user, userChoice])
```

Comment:
I'd change this to `showLeadingOption` as I always find using the opposite confusing :P 
Also... I don't think is necessary to memoize this condition.

Reply:
You can also group all the conditions
`const showLeadingOption = status === ProposalStatus.Active && !!user && !userChoice`

Reply:
At first I thought so but in this particular case, the 'show' has more than one condition. Instead, the 'hide' only has one condition and I felt it was easier to define it that way.

Regarding the Memo, it is true, it can be omitted :)